config/locales/en.yml

```diff
@@ -1232,6 +1257,7 @@ en:
     no_articles_wikidata: No items
     no_revisions: This editor has made no revisions.
     no_highlighting: "No text from the current version of the article is attributed to this editor. Check the page history to see their edits."
+    no_highlighting_wikidata: "No text from the current version of the item is attributed to this editor. Check the page history to see their edits."
```

Comment:
This feature does not work for Wikidata anyway — see https://github.com/WikiEducationFoundation/WikiEduDashboard/issues/4716 — so I don't think a variant message is necessary.